frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -380,6 +387,33 @@ local footerTextGeneratorMap = {
             end
         end
     end,
+    page_turning_inverted = function(footer)
+        local symbol_type = footer.settings.item_prefix
+        if symbol_type == "icons" or symbol_type == "compact_items" then
+            if G_reader_settings:isTrue("input_invert_page_turn_keys") or
+               G_reader_settings:isTrue("input_invert_left_page_turn_keys") or
+               G_reader_settings:isTrue("input_invert_right_page_turn_keys") or
+               G_reader_settings:isTrue("inverse_reading_order") then
+                return symbol_prefix.icons.page_turning_inverted
+            elseif footer.settings.all_at_once and footer.settings.hide_empty_generators then
+                return ""
+            else
+                return symbol_prefix.icons.page_turning_regular
+            end
+        else
+            local prefix = symbol_prefix[symbol_type].page_turning_inverted
+            if G_reader_settings:isTrue("input_invert_page_turn_keys") or
+               G_reader_settings:isTrue("input_invert_left_page_turn_keys") or
+               G_reader_settings:isTrue("input_invert_right_page_turn_keys") or
+               G_reader_settings:isTrue("inverse_reading_order") then
+                return T(_("%1 On"), prefix)
+            elseif footer.settings.all_at_once and footer.settings.hide_empty_generators then
+                return ""
+            else
+                return T(_("%1 Off"), prefix)
+            end
+        end
```

Comment:
May be reorganize all this to have the big "if any of the 4 settings is true" only once: move the symbol_type check inside that big if ?